stylesheets/styles.css

```diff
@@ -0,0 +1,219 @@
+html {
+	font-size: 16;
+	font-family: Karla,Helvetica,Arial;
+	/* footer hack */
+	height: 100%;
+}
+
+body {
+	/* footer hack */
+	position: relative;
+	padding-bottom: 40px;
+	margin-bottom: 0px;
+	min-height: 100%;
+}
+
+* {
+	box-sizing: border-box;
+}
+
+.clearfix {
+	clear: both;
+}
+
+header {
+	float: left;
+	display: block;
+    margin: 0 auto;
+	width: 25%
+
+}
+
+img {
+	max-width:100%;
+	max-height:100%;
```

Comment:
You actually don't need this max-height. Setting max-width will constrain the width and the rest of the image will scale with that.